lib/views/after_auth_screens/events/explore_events.dart

```diff
@@ -235,7 +234,14 @@ class ExploreEvents extends StatelessWidget {
     );
   }
 
-  // dropDownList returns a widget of a drop down list of events type.
+  /// Shows a list of dropdown taken from  `model` and `context`.
+  ///
+  /// params:
+  /// * [model] contains the events data
+  /// * [context] the overall context of UI
+  ///
+  /// returns:
+  /// * [Widget]: the dropdown
```

Comment:
Hey @literalEval I followed the guide on on docs.talawa.io 
but this is still giving the error.
![image](https://user-images.githubusercontent.com/65951872/224472389-3886a6c6-e215-494f-8b0a-f579abc61d20.png)

Can you find any mistakes here?

Reply:
@SiddheshKukade it's 

```
/// params:
/// * `model`: ....... 
```

and not

```
/// params:
/// * [model]: ....... 
```

basically replace `[` and `]` with `` ` ``

Reply:
Now it passes. I actually used that `[ ]` from the dart official docs.